### PR TITLE
appendMode true by default

### PR DESCRIPTION
**Pull request by AlvaroVega**
related with PRs: https://github.com/telefonicaid/iotagent-node-lib/pull/1111
https://github.com/telefonicaid/iotagent-node-lib/pull/1382

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -76,7 +76,8 @@ var config = {
     subservice: '/gardens',
     providerUrl: 'http://192.168.56.1:4041',
     deviceRegistrationDuration: 'P1M',
-    defaultType: 'Thing'
+    defaultType: 'Thing',
+    appendMode: true
```
Comment (fgalan) [inline config.js:80]: I understand that if a parameter is not in config.js is an implicit `false`, isn't it?

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -0,0 +1 @@
+- Make appendMode true by default
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: appendMode at general level (config.js / env var) changes default from false to true
```

File: config.js (hunk 1 above)
Comment (AlvaroVega) [reply config.js:80]: Yes. Internal behavior now has been changed in that sense

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -0,0 +1 @@
+- Fix: appendMode at general level (config.js / env var) changes default from false to true
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: appendMode at general level (config.js / env var) changes its default from false to true
```

**Review by fgalan — APPROVED** [review]
LGTM

File: config.js (hunk 1 above)
Comment (fgalan) [reply config.js:80]: NTC

**Comment by AlvaroVega** [comment]
Tests are broken after mege this PR and should be updated ASAP.

**Pull request merged**